### DISCUSSION TITLE
Lighthouse positioning issues

### DISCUSSION THREAD

**Discussion by r-bt** [bitcraze/discussions#1632] [Q&A]
I am trying to get a Crazyflie 2.0 to hover. I've connected to it with a Crazyradio PA and the cfclient. I've successfully installed the lighthouse deck and estimated the geometry. However, after taking off the Crazyflie begins to drift in all directions and on the Lighthouse tab of the cfclient the incorrect location is reported.

Here's a video of what's happening:

https://github.com/user-attachments/assets/43d3e6fd-98c7-43c7-86d9-080c45cbe016

**Comment by Bananazooka** [comment]
Hi, it looks like you're not receiving any data from the base stations. This could be caused by a number of things.... have you followed [this tutorial](https://www.bitcraze.io/documentation/tutorials/getting-started-with-lighthouse/) for calibration?
